Source/WebKit/Platform/IPC/Decoder.cpp

```diff
@@ -27,58 +27,52 @@
 #include "Decoder.h"
 
 #include "ArgumentCoders.h"
-#include "DataReference.h"
 #include "Logging.h"
 #include "MessageFlags.h"
 #include <stdio.h>
 #include <wtf/StdLibExtras.h>
 
 namespace IPC {
 
-static const uint8_t* copyBuffer(const uint8_t* buffer, size_t bufferSize)
+static const uint8_t* copyBuffer(DataReference buffer)
```

Comment:
Not new to this patch so you don't have to fix before landing but I don't think this should return a `const` pointer since it points to newly malloc'd memory that  the caller needs to free.

Reply:
Will fix.